tools/who_what_benchmark/whowhatbench/image2image.py

```diff
@@ -0,0 +1,143 @@
+import os
+from typing import Any, Union
+
+import datasets
+import pandas as pd
+from tqdm import tqdm
+from transformers import set_seed
+import torch
+import openvino_genai
+
+from .registry import register_evaluator
+from .text2image_evaluator import Text2ImageEvaluator
+
+from .whowhat_metrics import ImageSimilarity
+
+
+class Generator(openvino_genai.Generator):
```

Comment:
is it still will be needed after https://github.com/openvinotoolkit/openvino.genai/pull/1379 merge?

Reply:
merged, we can drop custom generator